com.github.TuxPusher.yaml

```diff
@@ -0,0 +1,22 @@
+app-id: com.github.TuxPusher
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: tuxpusher
+rename-icon: tuxpusher
```

Comment:
as it is right now you are missing the `rename-desktop-file` and `rename-appdata-file`

Reply:
added them back ;)